practice-app/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "practice-app-backend",
```

Comment:
this should not have backend tag I think @egirgin

Reply:
solved [here](https://github.com/bounswe/bounswe2020group4/pull/102/commits/81bbe81d712c483f306a053da35d38058fa688c9)